custom/icds/tasks.py

```diff
@@ -106,16 +105,24 @@ def get_language_code(user_id, telugu_user_ids, marathi_user_ids):
 
 
 @periodic_task(
-    run_every=crontab(hour=9, minute=0, day_of_week='mon'),
+    run_every=crontab(hour=9, minute=0),
     queue=settings.CELERY_PERIODIC_QUEUE,
     ignore_result=True
 )
-def run_weekly_indicators(phased_rollout=True):
+def run_user_indicators(phased_rollout=True):
     """
-    Runs the weekly SMS indicators Monday at 9am IST.
-    If it's the first week of the month, also run the monthly indicators.
+    Runs the weekly / monthly user SMS indicators at 9am IST.
```

Comment:
line 108 says `hour=9`, and I believe all of our servers have their timezone set to UTC, so I think this should say "3:30am IST"

Reply:
Sorry, I had my timezones switched. This should say "2:30pm IST"

Reply:
Celery workers run on IST for enikshay and icds so that downtime tasks coorespond more appropriately with midnight. There's a CELERY_TIMEZONE setting